.github/workflows/test-migrations-compatibility.yml

```diff
@@ -30,25 +30,21 @@ jobs:
           --health-retries 5
 
     steps:
-      - name: Checkout branch
-        uses: actions/checkout@v3
-
-      - name: Install system dependencies
-        run: apt-get install -y libpq-dev
```

Comment:
Checkout step is missing and `libpq-dev` is probably needed

Reply:
The deleted step has been restored for both files but as a separate step or perhaps `poetry` and `libpq-dev` installation should be treated as a one step